cosmos/x/evm/plugins/state/genesis.go

```diff
@@ -48,6 +48,9 @@ func (p *plugin) InitGenesis(ctx sdk.Context, ethGen *core.Genesis) {
 				p.SetState(address, k, v)
 			}
 		}
+		if account.Nonce != 0 {
```

Comment:
this if statement necessary?

Reply:
Maybe not, i think. I was debating whether to remove this part or not.

Reply:
ah okay, no problem to keep this.